.github/workflows/test.yaml

```diff
@@ -101,7 +115,7 @@ jobs:
         run: echo "GOPATH=${HOME}/go" >> $GITHUB_ENV
 
       - name: Set Go Path
-        run: echo "GO_JS_WASM_EXEC=${PWD}/test-wasm/go_js_wasm_exec" >> $GITHUB_ENV
+        run: echo "GO_JS_WASM_EXEC=${GOROOT}/misc/wasm/go_js_wasm_exec" >> $GITHUB_ENV
```

Comment:
This breaks the build.

Reply:
I added support for this before,
https://github.com/pion/.goassets/blob/55e0bbd19710d702327832ad2d0bc287b7c80d04/ci/.travis.yml#L91-L101
but it seems been dropped.

Reply:
Was broken in migration to GitHub actions :( https://github.com/pion/.goassets/issues/44